internal/controller/supervisorconfig/federation_domain_watcher.go

```diff
@@ -88,9 +89,9 @@ type federationDomainWatcherController struct {
 	oidcIdentityProviderInformer            idpinformers.OIDCIdentityProviderInformer
 	ldapIdentityProviderInformer            idpinformers.LDAPIdentityProviderInformer
 	activeDirectoryIdentityProviderInformer idpinformers.ActiveDirectoryIdentityProviderInformer
-
-	celTransformer *celtransformer.CELTransformer
-	allowedKinds   sets.Set[string]
+	githubIdentityProviderInformer          idpinformers.GitHubIdentityProviderInformer
```

Comment:
newline break here between informers and these other things should help see the diff better

Reply:
This file was merged in the other PR for the hack script, I'll be rebasing to pick it up again here.

Reply:
Hmm, my above comment seems to be in response to something else.

Are we always consistent with newlines in structs like this?  I'll add the space, but I wonder now common it is in our repo.